packages/core/src/components/slider/slider.tsx

```diff
@@ -49,20 +49,23 @@ export class Slider extends CoreSlider<ISliderProps> {
     private handle: Handle;
 
     protected renderFill() {
-        const { tickSize } = this.state;
+        const { tickSizeRatio } = this.state;
         const initialValue = clamp(this.props.initialValue, this.props.min, this.props.max);
 
-        let offset = Math.round((initialValue - this.props.min) * tickSize);
-        let size = Math.round((this.props.value - initialValue) * tickSize);
```

Comment:
it's cool that we don't need to `round` anymore, but i think we want to ensure a maximum number of decimal places via `number.toFixed(2)` to reduce sub-pixel rendering.

Reply:
Do you mean rounding the value of `offsetRatio`? I don't think that would accomplish much because the actual computed value in pixels doesn't have any restrictions on the number of decimal places. Here's an example of a round percentage yielding sub-pixel precision (shows inline styles and computed value from inspector):

![slider-subpixel](https://user-images.githubusercontent.com/205631/38383254-50ddb97a-390c-11e8-8511-e1d67457c395.png)

Unfortunately CSS `calc` provides no facilities for rounding the result. :cry:

Reply:
no i mean `${(offset * 100).toFixed(2)}%`

Reply:
sub-pixel renders are going to happen here for sure, but there's a limit to precision on the screen. it's definitely possible that my suggestion isn't really helpful?

Reply:
Well, at the very least we'll get inline styles that are easy on the eyes if anyone happens to be snooping around with inspect element. I'll make it happen :+1: